### PR TITLE
[B2BQUOTES-22] Send emails when quotes are created or updated

### PR DESCRIPTION
**Pull request by polishq**
#### What problem is this solving?

This PR updates the app so that appropriate users are notified when quotes are created or updated. Specifically:

- when a quote is created (using the "submit to sales rep" button), all users with the Sales Admin role will receive an email
- when a quote is updated, all users who have interacted with the quote (i.e. all emails that appear in the quote's update history) will receive an email

The PR provides default email templates for the two types of emails (`QuoteCreated` and `QuoteUpdated`). These will be automatically added to the store's account if they don't exist when the following GraphQL operations are executed: `getQuote`, `getQuotes`, `createQuote`

#### How to test it?

New version linked here: https://quotes--sandboxusdev.myvtex.com

1. Go to https://quotes--sandboxusdev.myvtex.com/admin/customers
2. Search for your vtex.com.br email and assign yourself as the Organization Admin of Aaaa Organization (in any cost center)
3. Search for your vtex.com email and assign yourself as the Sales Admin (in any organization and cost center)
4. Go to https://quotes--sandboxusdev.myvtex.com and login as your vtex.com.br email
5. Add products to your cart and then go to /b2b-quotes/create
6. Add a name and description to the quote and click `Submit to Sales Rep`
7. You should receive an email at your vtex.com address notifying that the quote was created
8. Log out of the storefront and log back in as your vtex.com email
9. Go to /b2b-quotes
10. Edit the quote that you just created, changing whatever you want, and save it
11. Both your vtex.com and vtex.com.br emails should receive notification that the quote was updated

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -46,6 +44,15 @@
...
     },
+    {
+      "name": "template-criar"
+    },
+    {
+      "name": "template-editar"
+    },
+    {
+      "name": "send-message"
+    },
```
Comment (arturmagalhaesjr) [inline manifest.json:55]: These policies are written in Portuguese, are they right?

File: node/clients/checkout.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ const CHECKOUT_ENDPOINT = 'api/checkout/pvt/configuration/orderForm'
 
 export const CHECKOUT_APP = 'b2b-quotes-graphql'
 
-export class Checkout extends JanusClient {
+export default class Checkout extends JanusClient {
```
Comment (arturmagalhaesjr) [inline node/clients/checkout.ts:14]: Why did you change to default export? Just understanding your changes

File: manifest.json (hunk 1 above)
Comment (polishq) [reply manifest.json:55]: Yes, you can see all available policies here: https://vtex.myvtex.com/admin/resources-list
Many have Portuguese names

**Review by arturmagalhaesjr — COMMENTED** [review]
I got only the e-mail was updated, the created didn't.
image
I'd add more information into e-mail subject like "<OrgName> ....", Also, I'd change the quote status message to upper case: REVISED.

Overall LGTM. Besides that, I'd ask a help @wender to review it once this PR is huge.

File: node/clients/checkout.ts (hunk 2 above)
Comment (polishq) [reply node/clients/checkout.ts:14]: Mostly to be consistent with the other clients. It affects how they're imported in `node/clients/index.ts`

**Comment by polishq** [comment]
> I got only the e-mail was updated, the created didn't. 
> I'd add more information into e-mail subject like "[OrgName] ....", Also, I'd change the quote status message to upper case: REVISED.
> 
> Overall LGTM. Besides that, I'd ask a help @wender to review it once this PR is huge.

Added your suggestions and relinked the new version of the app.

Only the sales admin(s) receives an email when the quote is created. I did receive one from you earlier:

```
Request for Quote
A new quote has been requested with the name "Test name $%@". Please use the link below to view and respond to the request.

Organization: Aaaa Organization
Cost Center: Button mash 2
Requested by: [artur.magalhaes@vtex.com.br](mailto:artur.magalhaes@vtex.com.br)
Notes: Test description AAAA AAAA
```

**Review by wender — APPROVED** [review]
